schema/monitoring_metric.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
maybe take a look at event_system.json  -- which is the current "proposed" (@Alpha) definition for the messages emitted by devices to report system metrics -- just to see how it's structured.  Not the same, of course, but just trying to make sure we're considering common constructs where appropriate.

Reply:
Ok I merged ptal

Reply:
Yeah, yesterday in our 1:1 I said I thought it might be a bit of a stretch to co-mingle the two. But we are visualizing it now. My other idea is to move just deviceId deviceNumId etc fields into a separate schema and refer to them in envelope.json.  (Not as a nested map, but as first level properties).  Then we share the definition in both places but break the association with messaging. WDYT?

Reply:
I toyed around with moving the fields into a resource.schema or similar but I couldn't get the $ref to work (and keep the depth/structure of envelope). I'm just going to keep it how it is for now.